project/plugins.sbt

```diff
@@ -0,0 +1,6 @@
+addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.5")
+addSbtPlugin("com.lucidchart" % "sbt-scalafmt" % "1.10")
```

Comment:
Lucidchart also has a neo scalafmt version and there is also a sbtplugin created by the maintainer of scalafmt http://scalameta.org/scalafmt/#sbt . Have these been considered?

Reply:
Yes I did look into that but this one seems to work more cleanly inside a complete sbt workflow